data/sql/updates/pending_db_world/rev_1650887143894550200.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1650887143894550200');
```

Comment:
delete this

Reply:
Ok, but why? don't you use this anymore?

Reply:
It doesn't exist anymore :)

Reply:
Just saying, the pending file comes with it so it should probably be dropped too.

Reply:
That was merged 3 hours ago, this was open 4 days ago